src/test/order_summary/OrderSummaryController.integration.test.ts

```diff
@@ -10,15 +10,26 @@ import * as apiClient from "../../client/api.client";
 import chai from "chai";
 import cheerio from "cheerio";
 import { InternalServerError, NotFound, Unauthorized } from "http-errors";
+import { BasketLink } from "../../utils/basket.util";
+import { BasketLinks } from "@companieshouse/api-sdk-node/dist/services/order/basket";
 
 let testApp;
 let sandbox = sinon.createSandbox();
+let getOrderStub;
+let getBasketLinksStub;
+let getBasketStub;
+
+const basketLink: BasketLink = {
+    showBasketLink: true,
+    basketWebUrl: "/basket",
+    basketItems: 1
+};
 
 describe("OrderSummaryController", () => {
     beforeEach((done) => {
         sandbox.stub(ioredis.prototype, "connect").returns(Promise.resolve());
         sandbox.stub(ioredis.prototype, "get").returns(Promise.resolve(signedInSession));
-        testApp = require("../../../src/app").default;
+        testApp = require("../../../src/app").default, basketLink;
```

Comment:
What is the purpose of adding `basketLink` to this `require` statement?

Reply:
Forgot to take this out